packages/blade/src/utils/mapProps/mapProps.native.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable guard-for-in */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {
+  accessibilityMap,
+  accessibilityStateKeys,
+  accessibilityValueKeys,
+  supportedRolesInNative,
+} from './a11yMap.native';
+import type { AccessibilityMap } from './a11yMap.native';
+
+function isAccessibilityStateProp(prop: string): boolean {
+  return accessibilityStateKeys.includes(prop);
+}
+
+function isAccessibilityValueProp(prop: string): boolean {
+  return accessibilityValueKeys.includes(prop);
+}
+
+const mapA11yProps = (props: AccessibilityMap): Record<string, unknown> => {
+  const newProps: Record<string, any> = {};
+
+  // loop through all the incoming props and map them
+  for (const key in props) {
+    const propKey = key as keyof AccessibilityMap;
+    const propValue = props[propKey];
+    const a11yKey = accessibilityMap[propKey];
+
+    // group accesibilityState prop for native
+    if (isAccessibilityStateProp(propKey)) {
```

Comment:
Arre anurag sir, we are just using this at one single place can just do `accessibilityStateKeys.includes(propKey)` in here only na? why to create one more function?